languages/go/config.json

```diff
@@ -27,7 +27,7 @@
         "uuid": "668463ec-4b73-40cf-b24d-2f3a4bb110f6",
         "concepts": ["bytes", "runes"],
         "prerequisites": [
-          "strings",
+          "strings-advanced",
```

Comment:
The `-advanced` suffix implies a hierarchy of concepts, which is [against the concept naming rules](https://github.com/exercism/v3/blob/master/docs/maintainers/determining-concepts.md#naming-concepts). Looking at the new `strings` exercise, it almost feels like that exercise is more about `string-formatting` than `strings`? Or perhaps `strings-advanced` could be `strings-package` or something like that?